src/shared/discover-image.c

```diff
@@ -1326,6 +1338,7 @@ static const char* const image_class_table[_IMAGE_CLASS_MAX] = {
         [IMAGE_MACHINE] = "machine",
         [IMAGE_PORTABLE] = "portable",
         [IMAGE_EXTENSION] = "extension",
+        [IMAGE_CONFIGURATION] = "configuration",
```

Comment:
maybe name this "syscfg", i.e. the same way as the directories we place these things in